src/controllers/commands_controller.js

```diff
@@ -208,4 +220,33 @@ CommandsController.post('/', async (request, response) => {
   });
 });
 
+const wss = new WebSocket.Server({port: CommandUtils.wsSettings.port});
```

Comment:
This should use `CommandsController.ws('/', function(ws) {});` so that it doesn't have to do its own JWT checking and connection management. I also think that listening on a port like this doesn't work with the tunneling system.